include/mxnet/ndarray.h

```diff
@@ -789,15 +810,22 @@ class NDArray {
     // size is the number of bytes
     void CheckAndAlloc(uint64_t dbytes) {
       CHECK_EQ(kDefaultStorage, storage_type)
-              << "CheckAndAlloc(dbytes) is not intended for kDefaultStorage";
+          << "CheckAndAlloc(dbytes) is not intended for kDefaultStorage";
```

Comment:
That was an incorrect error msg: `CheckAndAlloc(dbytes) is **only** intended for kDefaultStorage`